banzai_floyds/tests/test_binning.py

```diff
@@ -0,0 +1,31 @@
+from banzai_floyds.utils.binning_utils import bin_data
+from collections import namedtuple
+import numpy as np
+
+
```

Comment:
This test checks that we have the expected number of pixels in each of our bins after binning. It has been quite difficult to define tests to ensure that our binning is behaving correctly for the full dataset.